membership.md

```diff
@@ -0,0 +1,170 @@
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Kubeflow GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in a repository | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a repository | [OWNERS] file approver entry|
+
+## New contributors
+
+[New contributors] should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the principles in this document, familiarity with project organization, roles, policies, procedures, conventions, etc., and technical and/or writing ability. Role-specific expectations, responsibilities, and requirements are enumerated below.
+
+## Member
+
+Members are *[continuously active]* contributors in the community. They can have issues and PRs assigned to them and tests are automatically run for their PRs. Members are expected to remain active contributors to the community.
+
+**Defined by:** Member of the Kubeflow GitHub organization
+
+### Requirements
+
+- Enabled [two-factor authentication] on their GitHub account
+- Have made **at least** 2-3 contributions to the project or community.
+- Have read the [contributor guide]
+- Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
+- **[Open an issue][membership request] against the kubeflow/internal-acls repo**
+  - Ensure your sponsors are @mentioned on the issue
+- **Open a pull request against the kubeflow/internal-acls repo**
+  - Complete every item on the checklist ([preview the current version of the template][membership template])
+  - Make sure that the list of contributions included is representative of your work on the project.
+- Have your sponsoring reviewers reply confirmation of sponsorship
+- Once your sponsors have responded, your request will be reviewed by the Kubeflow team. Any missing information will be requested
+
+### Responsibilities and privileges
+
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- Members can do `/lgtm` on open PRs.
+- They can be assigned to issues and PRs, and people can ask members for reviews with a `/cc @username`.
+- Tests can be run against their PRs automatically. No `/ok-to-test` needed.
+- Members can do `/ok-to-test` for PRs that have a `needs-ok-to-test` label, and use commands like `/close` to close PRs as well.
```

Comment:
Maybe we should also add something from here: https://github.com/kubernetes/community/blob/master/community-membership.md#requirements

- Subscribed to [kubeflow-discuss@googlegroups.com](https://groups.google.com/a/kubernetes.io/group/dev)